### ISSUE TITLE
bigint as `id` field seems unqueryable

### ISSUE THREAD

**Issue by shrugs** [0xOlias/ponder#248]
using a `BigInt!` as an id (in the erc721 example, for example) seems to make the `token(id)` field always return null, even if passing a valid id. it also makes relationships nulled

changing to a `String!` fixed this for me, and i simply cast bigints to String when CRUDing entities

i expect that using a BigInt as id is desired, but whatever is powering the graphql side of things doesn't seem to be able to look up entities by id if that id is a bigint

**Comment by 0xOlias** [comment]
Was able to repro - working on a fix now!

**Comment by 0xOlias** [comment]
> it also makes relationships nulled

Could you elaborate / share an example for this? Not sure which relationship/direction you're referring to. I was able to repro the `token(id: "123")` using this query in the `token-erc721` example:

```graphql
{
  token(id: "6796") {
    id
  }
}
```
returns
```json
{
  "data": {
    "token": null
  }
}
```
which is a bug.

**Comment by shrugs** [comment]
sorry i didn't provide a repro, was in the flow. 

if an entity has a relationship like

```gql
type Account @entity {
  id: String!
  tokens: [Token!]! @derivedFrom(field: "owner")
}

type Token @entity {
  id: BigInt!
  owner: Account!
  transferEvents: [TransferEvent!]! @derivedFrom(field: "token")
}

type TransferEvent @entity {
  id: String!
  from: Account!
  to: Account!
  token: Token!
  transactionHash: String!
  timestamp: Int!
}
```

the reverse lookup isn't possible:


```gql
{
  transferEvents(where:{token:"1"}) {
    id
    token {
      id
    }
  }
}
```

returns

```json
{
  "data": {
    "transferEvents": [
      {
        "id": "0xee0d6104cff1b2020f357d3621cf0a8ff54ff6fa5dd0545f17052a7552a872ff-0x0",
        "token": null
      }
    ]
  }
}
```

this is _probably_ the same issue, i.e. the engine looking up an entity by BigInt id and returning null. querying one-to-many relationships works, though:


```gql

# works
query Account($id: ID!) {
  account(id: $id) {
    tokens {
      id
    }
  }
}
```

**Comment by 0xOlias** [comment]
Got it, thanks a bunch! I just found a handful of `BigInt` related bugs in the schema/GraphQL resolver generation code. 😅

Thanks for reporting, working on fixes now.

**Comment by 0xOlias** [comment]
@shrugs Should be fixed in [`0.0.70`](https://github.com/0xOlias/ponder/releases/tag/%40ponder%2Fcore%400.0.70).
